parser/prism/Translator.h

```diff
@@ -19,6 +19,7 @@ class Translator final {
     // but don't have explicit ownership over it. We take a temporary reference to it, but we can't
     // escape that scope, which is why Translator objects can't be copied, or even moved.
     core::GlobalState &gs;
+    uint16_t uniqueCounter = 1;
```

Comment:
See the `$` values in the test expectations, to see how this is used.

Reply:
p.s. I noticed that parts (all of?) the rest of the repo uses a suffixed `_` for member variables. We should come back and change this, as well as any other such stylistic changes.